### DISCUSSION TITLE
Allow Displaying 2FA Status on the Profile / Organisation Member List?

### DISCUSSION THREAD

**Discussion by ChristianTackeGSI** [community/community#30062] [Other Feature Feedback, Questions, & Ideas] [answered]
[By the end of 2023, 2FA will be required for developers](https://github.blog/2022-05-04-software-security-starts-with-the-developer-securing-developer-accounts-with-2fa/), so the status should be shown somewhere?

I see a few places, where this could be useful:
* Just show you're ahead of things and already activated 2FA!
* Check people's status you would like to collaborate with
* In an ORG: View the status in the member list before enforcing 2FA for the whole ORG.

What do you think?

**Comment by ichenglin** [comment, marked as answer]
You could navigate to **`https://github.com/orgs/ {ORGANIZATION_NAME} /people`**
to view the **`2FA`** status of all the **members in the organization**.

![2fa](https://user-images.githubusercontent.com/41904540/185732109-6073e687-f1e0-4112-9615-3fc547eac085.png)

**Reply by ChristianTackeGSI** [reply]
Wow! Didn't notice this, because I was only looking at ORGs where I am not an Owner.
Just checked an ORG, where I have Owner-status, and there it is!

So, my point about ORGs is already solved, good!

Is 2FA considered a "private" status? If so: Why?
Does it reveal too much "attack surface"?

**Reply by ichenglin** [reply]
I **don't** think revealing the **`2FA`** status is that much of a **security concern**,
yet there's **no point in showing** it to the public either.

**Reply by ChristianTackeGSI** [reply]
Well, it could be shown to other members of the same ORG?
